src/nanoemoji/nanoemoji.py

```diff
@@ -411,11 +411,15 @@ def write_bitmap_builds(
     bitmap_builds: Set[Path],
     nw: NinjaWriter,
     clipped: bool,
+    has_picosvgs: bool,
     master: MasterConfig,
 ):
     os.makedirs(str(bitmap_dir()), exist_ok=True)
     for svg_file in master.sources:
-        dest = bitmap_dest(svg_file)
+        if has_picosvgs:
+            dest = picosvg_dest(clipped, svg_file).with_suffix(".png")
+        else:
+            dest = bitmap_dest(svg_file)
```

Comment:
it was intentional, I wanted to place the bitmap for the hybrid font next to their respective picosvgs -- it does the job, but you're right it's better to have explicit inputs and keep bitmaps in their own directories.

Reply:
now I remember why I didn't go the explicit input *.png route. The write_font script doesn't simply take a list of svgs (it used to), but a glyphmap.csv file with svg filenames mapped to glyph names and optional codepoints. You don't want to add additional rows to this glyphmap.csv for the *.png input files, for that would risk having inconsistent mappings for the svg and png glyphs; but if you add an extra column for the .png filename, then most of the time for vector-only fonts, that column will be empty (two ugly commas next to one another separated by whitespace at best)... Not just the default write_glyphmap need changing, but also any client-provided glyphmap-generator would have to change to allow outputing the additional optional column for the .png filenames. Sure, we _can_ do it.
The alternative of having a flag that tells write_font how to turn an svg filename into the respective png, like changing the prefix to a different directory and replace the .svg suffix with .png (so that pngs can live in their separate "bitmap" build subdirectory), would not work reliably because of the dynamic disambiguation logic you added for avoiding clashes of sources with the same basename.